src/Core/Style.js

```diff
@@ -919,17 +958,16 @@ class Style {
      * @param {Boolean} canBeFilled - true if feature.type == FEATURE_TYPES.POLYGON.
      */
     applyToCanvasPolygon(txtrCtx, polygon, invCtxScale, canBeFilled) {
-        const context = this.context;
         // draw line or edge of polygon
         if (this.stroke) {
             // TO DO add possibility of using a pattern (https://github.com/iTowns/itowns/issues/2210)
```

Comment:
should we check for stroke properties too, like for fill ?

Reply:
`this.stroke` and `this.fill` are never undefined as the both have properties with default value. The previously done check were then useless.
In the end state of this PR both thes were removed for more specific tests.